airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/CDCMySqlDatatypeAccuracyTest.java

```diff
@@ -26,4 +27,11 @@ protected Database setupDatabase() {
     return testdb.getDatabase();
   }
 
+  // Temporarily disable this test since it's causing trouble on GHA.
+  @Override
+  @Test
+  public void testDataContent() {
```

Comment:
Are we worried about loss of coverage here?

Reply:
a little bit - so want to hear your opinion too!

Arguing points are:
1. The parent class defined all test cases but did not specify expected values - to me this sounds like we are just to test DataTypes only; which should be covered by `testDataType()` function I think
2. Local run passed. I think we want to unblock the release for now if it's safe.

Reply:
Discussed offline, fine to disable for now